### PR TITLE
Fixes #15684 - foreman-rake without options raise an error

### PR DESCRIPTION
**Pull request by ranjan**

### CODE REVIEW

File: script/foreman-rake [hunk 1]
```diff
@@ -10,7 +10,9 @@ die() {
 cd ~foreman || die "Cannot change to foreman home directory"
 [ -f Gemfile ] && BUNDLER_CMD="bundle exec"
 CMD="$BUNDLER_CMD $RAKE_CMD"
-if [ $USERNAME = foreman ]; then
+if [ $# -eq 0 ]; then
+  foreman-rake -h; exit 0;
```
Comment (iNecas) [inline script/foreman-rake]: how about using `foreman-rake -D -A` instead: this prints all available tasks (not that all would be needed for us. Maybe the foreman-rake should have it's own help just with relevant commands, but that's probably for different issue.

**Comment by iNecas** [comment]
Tested and works well: one question about the help output inline

File: script/foreman-rake (hunk 1 above)
Comment (domcleal) [inline script/foreman-rake]: It shouldn't call `foreman-rake` recursively, there's no need, it should be calling `rake` directly as it does below. (And if it did, it should be using `$0`.)

Comment (ranjan) [inline script/foreman-rake]: perfect, Thanks @domcleal for pointing out the issue. Will update the script with
$CMD -h; exit 0;

**Comment by ranjan** [comment]
@iNecas "foreman-rake -D" also includes help for tasks  which is not required and "-h" only prints options related to foreman-rake so "-h" looks ok here.

**Comment by iNecas** [comment]
my point was the user of foreman-rake is more intersted into the tasks that the rake options. Anyway, that's more of a question of how the help for the foreman-rake should look like than the core fix of this PR, so I'm ok with it

**Comment by ranjan** [comment]
[test] failure looks unrelated to the change

**Comment by dLobatog** [comment]
Merged as aa9f27e339580ac91f7ebc42c2ca31aea6c8a19d, thanks @ranjan!

**Pull request closed without merging**